spec/lib/activitypub/tag_manager_spec.rb

```diff
@@ -7,17 +7,50 @@
 
   subject { described_class.instance }
 
+  let(:domain) { "#{Rails.configuration.x.use_https ? 'https' : 'http'}://#{Rails.configuration.x.web_domain}" }
+
+  describe '#public_collection?' do
```

Comment:
This is just coverage for an already existing method here, and not something from this PRs changes, right?

Reply:
Yes